Expensify/App
# fix Modal animations

Please review.

### Details
Details can be read https://github.com/Expensify/Expensify.cash/issues/2335#issuecomment-820618607

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes #2335 

### Tests / QA Steps
1. Click the search icon to open the Search Modal.
2. Open all the available models from the FAB button.
3. Open the User Details / Group participants modal.
4. Open the User Settings Modal.
5. **For all the modals enter and exit animation should be smooth and consistent.**
6. Open Emoji Picker on Mobile devices and Web they should be animated.

### Tested On


### Screenshots


#### Web
   



https://user-images.githubusercontent.com/24370807/116021093-205ff780-a665-11eb-9635-c64beceb9567.mp4

 


#### Mobile Web


https://user-images.githubusercontent.com/24370807/116024431-cf073680-a66b-11eb-93c0-a452dbd8fdff.mp4


#### Desktop



https://user-images.githubusercontent.com/24370807/116080015-2af6ad00-a6b6-11eb-9960-4e2fef0b2232.mp4



#### iOS

 Not able to record it as I have MAC VM but the animation on IOS works smoothly.
#### Android


https://user-images.githubusercontent.com/24370807/116021378-b7c54a80-a665-11eb-94d8-5ed0de1cf36d.mp4
